nemo/collections/asr/data/audio_to_text_lhotse_prompted.py

```diff
@@ -56,12 +55,21 @@ def __init__(
     def __getitem__(self, cuts: CutSet) -> tuple[torch.Tensor, torch.Tensor, torch.Tensor, torch.Tensor]:
         audio, audio_lens, cuts = self.load_audio(cuts)
 
-        tokens = self.prompt_format_fn(cuts, self.tokenizer, self.inference)
+        tokens, prompt_tokens = self.prompt_format_fn(cuts, self.tokenizer, inference=self.inference)
```

Comment:
@pzelasko we need to separate prompt_fn from cuts / Lhotse for simpler usage. Lhotse can wrap the generic class/function